src/pages/UserProfile.js

```diff
@@ -0,0 +1,80 @@
+import {useState, useEffect} from 'react';
+import {div, h, hr} from 'react-hyperscript-helpers';
+import {PageHeading} from '../components/PageHeading';
+import {Notification} from '../components/Notification';
+import {FormField, FormFieldTypes, FormValidators} from '../components/forms/forms';
+import {User} from '../libs/ajax';
+import {NotificationService} from '../libs/notificationService';
+import { Notifications} from '../libs/utils';
+
+export default function ResearcherProfile(props) {
+  const [profile, setProfile] = useState({
+    profileName: '',
+    email: undefined,
+    id: undefined
+  });
+
+  const [notificationData, setNotificationData] = useState();
+
+  useEffect(() => {
+    const init = async () => {
+      try {
+        await getResearcherProfile();
+
+        setNotificationData(await NotificationService.getBannerObjectById('eRACommonsOutage'));
+      } catch (error) {
+        Notifications.showError({text: 'Error: Unable to retrieve user data from server'});
+      }
+    };
+
+    init();
+  }, [props.history]);
```

Comment:
I don't think you need this effect to be dependent on props.history - there's nothing in the code now that modifies it. We may need it in future iterations, but let's do that when we need it.